src/generated/.openapi-generator/FILES

```diff
@@ -1,113 +1,31 @@
 .openapi-generator-ignore
-apis/ApiKeysApi.ts
-apis/AppMetadataSchemaApi.ts
 apis/AppsApi.ts
-apis/EmailProvidersApi.ts
-apis/EmailTemplateApi.ts
-apis/EmailTestApi.ts
-apis/EventsApi.ts
 apis/MagicLinkApi.ts
-apis/TokensApi.ts
```

Comment:
The TokensApi shouldn't be deleted.

Reply:
It is used here: https://github.com/passageidentity/passage-node/blob/DX-44-node-sdk-work/src/classes/User.ts#L250C7-L250C7